examples/pipelines/simple_pipeline/config/components_config.py

```diff
@@ -23,8 +23,8 @@ class ImageFilterConfig:
         MIN_WIDTH (int): Minimum width for each image.
     """
 
-    MIN_HEIGHT = 200
-    MIN_WIDTH = 200
+    MIN_HEIGHT = 600
```

Comment:
what's the reason for bumping this up? we might be filtering everything in this case.

Reply:
The image filtering component wasn't filtering out any images, so I just set this higher to make sure the resulting dataframe contains less images than the original one, to see whether the index is properly updated

Reply:
In this, out of the 833 images, we are left with 551 images.

Reply:
alright now it's more clear. Thanks!